### PR TITLE
Add substring search

### PR DESCRIPTION
**Pull request by fazd**
I Add the program substring search resolving issue #68

### CODE REVIEW

File: STRING/C++/StringSearch.cpp [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                c_new[p[i]] = c_new[p[i-1]];
+            }
+            else{
+                c_new[p[i]] = c_new[p[i-1]]+1;
+            }
+        }
+        c = c_new;
+        len*=2;
+    }
+    /*for(int i = 0; i < n; i++){
```
Comment (GauravWalia19) [inline STRING/C++/StringSearch.cpp:83]: Remove this commented code.

File: STRING/C++/StringSearch.cpp [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    return p;
+}
+ 
+vector<int> pr;
+string str,s;
+ 
+ 
+bool searchWord(int start, int end){
+    int mid = start + (end- start)/2;
+    //cout<<"start: "<<start<<" end: "<<end<<" mid: "<<mid<<"\n";
```
Comment (GauravWalia19) [inline STRING/C++/StringSearch.cpp:96]: Remove this one also

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]

**Comment by fazd** [comment]
Checked, I remove both comments

File: STRING/C++/StringSearch.cpp (hunk 2 above)
Comment (fazd) [reply STRING/C++/StringSearch.cpp]: Done

File: STRING/C++/StringSearch.cpp (hunk 1 above)
Comment (fazd) [reply STRING/C++/StringSearch.cpp]: Done

**Review by GauravWalia19 — APPROVED** [review]

**Pull request merged**